src/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

```diff
@@ -1922,6 +1961,10 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 *  The `getList` method has been refactored to retrieve the `is_subscribed` extension attribute. Previously, you could not retrieve this attribute when fetching a customer entity through the API. _Fix submitted by enriquei4 in pull request [25311](https://github.com/magento/magento2/pull/25311)_. [GitHub-13252](https://github.com/magento/magento2/issues/13252)
 
+<!--- ENGCOM-6500-->
+
+*  Updating customer information through `rest/all/V1/customers/:id` resets `group_id` no longer resets the `group_id` unnecessarily. Previously, when you updated a customer through REST, the customer group was changed to 1 if the body did not contain a customer group. _Fix submitted by MaxRomanov4669 in pull request [25958](https://github.com/magento/magento2/pull/25958)_. [GitHub-14663](https://github.com/magento/magento2/issues/14663)
```

Comment:
```suggestion
*  Updating customer information through `/V1/customers/:id` no longer resets the `group_id` unnecessarily. Previously, when you updated a customer through REST, the customer group was changed to 1 if the body did not contain a customer group. _Fix submitted by MaxRomanov4669 in pull request [25958](https://github.com/magento/magento2/pull/25958)_. [GitHub-14663](https://github.com/magento/magento2/issues/14663)
```

This passage needs to be updated in Commerce as well.

Reply:
Updated in both sets of notes